patterns/blueprint-vpc-lattice/cluster2/secrets.sh

```diff
@@ -0,0 +1,36 @@
+##!/bin/bash
+DBHOST="$(terraform output -raw postgres_host)"
+DBUSER="$(terraform output -raw postgres_username)"
+DBPASSWORD="$(terraform output -raw postgres_password)"
+DBPORT="$(terraform output -raw postgres_port)"
+DBNAME="$(terraform output -raw postgres_db_name)"
+DBSCHEMA=analytics
+
+
+CLUSTER_2=cluster2
+AWS_DEFAULT_REGION=$(aws configure get region)
+AWS_ACCOUNT_NUMBER=$(aws sts get-caller-identity --query "Account" --output text)
+
+
+export CTX_CLUSTER_2=arn:aws:eks:$AWS_DEFAULT_REGION:${AWS_ACCOUNT_NUMBER}:cluster/$CLUSTER_2
+
+
+kubectl apply --context="${CTX_CLUSTER_1}" -f ./$CLUSTER_1/gateway-lattice.yml
+kubectl apply --context="${CTX_CLUSTER_2}" -f ./$CLUSTER_2/gateway-lattice.yml
```

Comment:
we can't apply because we didn't run yet the aws eks update-kubeconfig --name $CLUSTER_2 --region $AWS_DEFAULT_REGION command within the instructions

Reply:
in the instruction we have done
```
cd cluster2
chmod +x secrets.sh && ./secrets.sh
```

so there is nothing in ./$CLUSTER_2, we are already in this directory

Reply:
i have applied them